src/core/components/tabs/tabsContent/tabsContent.tsx

```diff
@@ -1,12 +1,25 @@
-import { TabsContent as RadixTabsContent, type TabsContentProps as RadixTabsContentProps } from '@radix-ui/react-tabs';
+import { TabsContent as RadixTabsContent } from '@radix-ui/react-tabs';
+import { type ComponentProps } from 'react';
 
-export interface ITabsContentProps extends RadixTabsContentProps {
+export interface ITabsContentProps extends ComponentProps<'div'> {
     /**
      * Value linking Tabs.Content to its corresponding Tabs.Trigger
      */
     value: string;
+    /**
+     * When `true`, the content will stay mounted even when inactive.
+     */
+    forceMount?: true;
 }
 
+/**
+ * `TabsContent` component
+ *
+ * This component is based on the Radix-UI tabs content implementation.
+ * An exhaustive list of its properties can be found in the corresponding Radix primitive
+ * [documentation](https://www.radix-ui.com/primitives/docs/components/tabs#content).
+ */
```

Comment:
This comment is not needed now as we're not extending the radix-ui properties. The comment should also be updated/removed from the other components

Reply:
✅ https://github.com/aragon/ods/pull/218/commits/53f11746677dbc94c6e87a573d869fbdb78b164c